mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -14,42 +14,53 @@ To get more information about Environments, see:
 
 * [Cloud Composer documentation](https://cloud.google.com/composer/docs)
 * [Cloud Composer API documentation](https://cloud.google.com/composer/docs/reference/rest/v1beta1/projects.locations.environments)
-* How-to Guides (Cloud Composer 1)
-  * [Creating environments](https://cloud.google.com/composer/docs/how-to/managing/creating)
-  * [Scaling environments](https://cloud.google.com/composer/docs/scale-environments)
-  * [Configuring Shared VPC for Composer Environments](https://cloud.google.com/composer/docs/how-to/managing/configuring-shared-vpc)
 * How-to Guides (Cloud Composer 2)
   * [Creating environments](https://cloud.google.com/composer/docs/composer-2/create-environments)
   * [Scaling environments](https://cloud.google.com/composer/docs/composer-2/scale-environments)
   * [Configuring Shared VPC for Composer Environments](https://cloud.google.com/composer/docs/composer-2/configure-shared-vpc)
+* How-to Guides (Cloud Composer 3)
+  * [Creating environments](https://cloud.google.com/composer/docs/composer-3/create-environments)
+  * [Scaling environments](https://cloud.google.com/composer/docs/composer-3/scale-environments)
+  * [Change environment networking type (Private or Public IP)](https://cloud.google.com/composer/docs/composer-3/change-networking-type)
+  * [Connect an environment to a VPC network](https://cloud.google.com/composer/docs/composer-3/connect-vpc-network)
 * [Apache Airflow Documentation](http://airflow.apache.org/)
 
-We **STRONGLY** recommend you read the [GCP
```

Comment:
Do we want to remove this warning? I believe it still applies for the most part?